transcom/mymove
# B 20290 int add amount reimbursement

## [B-20290](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A972193)

## Summary

This story adds Actual SIT Reimbursement field to the Review Documents page for storage moving expenses. The value should be the smaller of the Government SIT Cost and the Amount Requested. The Actual SIT Reimbursement amount will adjust as SIT Cost and Amount Requested are updated. The Actual SIT Reimbursement amount is saved into a new DB column in the moving_expenses table and will be loaded into the SSW for shipments that have a storage type moving expense. The SSW was also updated to use the Actual SIT Reimbursement amount for PPM shipments with storage moving expense in the Member-paid SIT field.

![image](https://github.com/transcom/mymove/assets/149000222/530067ae-e6dc-4487-93e7-6206f1bd4f92)


## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Login as a Customer
2. Create a PPM shipment and make sure you enter Yes when asked if you plan to use storage
3. Submit the shipment
4. Login as a Service Counselor
5. Approve the shipment
6. Login as the customer
7. Upload PPM documents and fill out the first few pages
8. Make sure you add a Storage type moving expense to the shipment
9. Submit the documents
10. Login as the Service Counselor
11. Go to the PPM Closeout Queue select the shipment
12. Click on the Review Documents button
13. You should see the Actual SIT Reimbursement which is the lower of the Government SIT Cost and Amount Requested
14. Adjust the location or other SIT items so the cost updates and also adjust Amount Requested verify Actual SIT Reimbursement is the lower of the two amounts
15. Accept and approve the documents
16. Click the download link for the Payment Packet
17. Look at the Member-paid SIT field and verify the amount is the same as was shown on the Actual SIT Reimbursement displayed on the Review Documents page

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
